libs/ui-lib-tests/cypress/fixtures/custom-manifests/index.ts

```diff
@@ -0,0 +1,22 @@
+import { customManifestsCluster } from './1-cluster-created';
+import { clusterReadyBuilder } from '../create-sno/5-cluster-ready';
+import { isoDownloadedClusterBuilder } from '../create-sno/2-iso-downloaded';
+import { hostDiscoveredBuilder } from '../create-sno/3-host-discovered';
+import { hostRenamedBuilder } from '../create-sno/4-host-renamed';
+import { customManifest, customManifestContent } from './manifests';
+
+const isoDownloadedCluster = isoDownloadedClusterBuilder(customManifestsCluster);
+const hostDiscoveredCluster = () => hostDiscoveredBuilder(isoDownloadedCluster);
+const hostRenamedCluster = () => hostRenamedBuilder(hostDiscoveredCluster());
```

Comment:
Why are these two wrapped in arrow functions?

Reply:
copy&paster error